R/transformCounts.R

```diff
@@ -52,22 +52,8 @@
 #'
 #' \itemize{
 #' 
-#' \item 'alr': Additive log ratio (alr) transformation, please refer to 
-#' \code{\link[vegan:decostand]{decostand}} for details.
-#' 
-#' \item 'chi.square': Chi square transformation, please refer to 
-#' \code{\link[vegan:decostand]{decostand}} for details.
-#' 
-#' \item 'clr': Centered log ratio (clr) transformation, please refer to 
-#' \code{\link[vegan:decostand]{decostand}} for details.
-#'
-#' \item 'frequency': Frequency transformation, please refer to 
-#' \code{\link[vegan:decostand]{decostand}} for details.
-#' 
-#' \item 'hellinger':  Hellinger transformation, please refer to 
-#' \code{\link[vegan:decostand]{decostand}} for details.
-#' 
-#' \item 'log':  Logarithmic transformation, please refer to 
+#' \item \code{alr, chi.square, clr, frequency, hellinger, log, normalize, pa, rank, rclr
```

Comment:
The code formats the values incorrectly (it shows that the list is one value). Instead, you have to format values one by one or just like 'alr', 'frequency', ...